Statut_HSP.tex

```diff
@@ -168,19 +174,47 @@ \section{Członkowie Stowarzyszenia}
   regularnego opłacania składek.
 \end{enumerate}
 
-\myparagraph{§ 14}
+\myparagraph{§ 12.NZ.1}
+\begin{enumerate}
+\item
+  Prawa i obowiązki każdego członka nadzwyczajnego z osobna definiuje zarząd.
+\item
+  Prawa i obowiązki członka nadzwyczajnego mogą być zmienione na jego wniosek po zatwierdzeniu przez zarząd.
+\item
+  Prawa członka nadzwyczajnego muszą zawierać się w zbiorze praw członka zwyczajnego.
+\item
+  Członek nadzwyczajny musi mieć obowiązek regularnego opłacania składek członkowskich.
+\item
+  Członek nadzwyczajny musi mieć obowiązek przestrzegania statutu i uchwał władz stowarzyszenia.
+\item 
+  Członek nadzwyczajny musi nie posiadać biernego oraz czynnego prawa wyborczego.
+\item
+  Członek zwykły może w każdym momencie zostać członkiem nadzwyczajnym informując w formie pisemnej zarząd.
```

Comment:
To się nie zgadza, bo jeżeli Zarząd może zmienić subset praw/obowiązków, to sama deklaracja nic nie zmienia - zostajesz członkiem nadzwyczajnym z takimi samymi prawami jak zwyczajny. 

Moze chodziło o to, że członek nadzwyczajny może za deklaracją stać się zwyczajnym? Wtedy jest to warunek wyjścia z tego stanu.

Reply:
rel, zmieniam